source.mak

```diff
@@ -113,7 +114,7 @@ SOURCES = $(MAIN_SOURCES) $(PARSER_SOURCES) $(DEBUG_SOURCES)
 
 ENVIRONMENT_HEADERS = e_msoft.h
 
-ENVIRONMENT_SOURCES = 
+ENVIRONMENT_SOURCES =
```

Comment:
Although I agree with the change, since only white spaces are modified I would prefer not to have this included in the patch.